hugo/static/.htaccess

```diff
@@ -43,16 +43,17 @@ RewriteRule ^[A-Za-z]/[A-Za-z][0-9][0-9]/([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9
 # This way there is only one page. Should maintain for backward compatibility for some time after August 2019.
 RewriteRule ^papers/[A-Za-z]/[A-Za-z][0-9][0-9]/([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9][0-9])/?$ https://www.aclweb.org/anthology/$1$2-$3 [R=301,L]
 
-# Volume pages (e.g., redirect /W19-75 to /volumes/W19-75)
-RewriteRule ^([A-Za-z])([0-9][0-9])\-([0-9][0-9])/?$ https://www.aclweb.org/anthology/volumes/$1$2-$3 [R=301,L]
-
 #
 # INTERNAL RETRIEVALS
 #
 # These rules take valid external URLS (which are often virtual targets) and retrieve their target.
 # They are invisible to the outside.
 #
 
+# Volume pages (e.g., redirect /W19-75 to /volumes/W19-75)
+RewriteRule ^([A-Za-z][0-9][0-9]\-[0-9]{1,2})/?$ /anthology/volumes/$1 [L,NC]
+RewriteRule ^(\d{4}\.[a-zA-Z\d]+\-[a-zA-Z\d]+)/?$ /anthology/volumes/$1 [L,NC]
+
```

Comment:
This identifies old-style and new-style volume URLs, and loads the right page. This allows volumes to have canonical URLs, just like papers.

Reply:
Should we be consistent with using `[0-9]` vs. `\d`? Or is there a reason to have sometimes one, sometimes the other?

Reply:
No, this is just chance. Consistency would be good. I'll use `\d` unless someone objects.